Extension/src/LanguageServer/client.ts

```diff
@@ -1856,9 +1856,11 @@ export class DefaultClient implements Client {
         if (!currentProvider.isReady) {
             return;
         }
+
         await Promise.all([
             this.clearCustomConfigurations(),
-            this.handleRemoveAllCodeAnalysisProblems(),
+            this.handleRemoveAllCodeAnalysisProblems()]);
```

Comment:
Is the formatting okay? The formatter doesn't make any changes.

Reply:
Formatting seems okay to me...I don't see any existing code using the previous formatting.